PublicLedger.test.ts

```diff
@@ -29,7 +29,7 @@ test('PublicLedger', async () => {
   fetchMock.mockResponse(JSON.stringify({access_token:PUBLIC_TOKEN}));
   const {result, waitForValueToChange} = renderPublicLedgerHook(() => { return 'we-have-a-token'; });
   expect(fetchMock.mock.calls.length).toEqual(1);
-  expect(result.current).toEqual(null);
+  expect(result.current).toEqual(undefined);
```

Comment:
I was getting exceptions thrown when running these tests on `"@testing-library/react-hooks": "^3.3.0"`, that I also reproduced on master without any other changes.

After upgrading the testing library to `5.0.3`, the tests worked, but now expect `undefined` instead of `null`.